cue/schemas/panels/time-series-table/time-series-table.cue

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 The Perses Authors
```

Comment:
Should we change to 2024?

Reply:
everywhere ? yeah probably... conflicts with pending PR incoming :smile:

Reply:
I will at least change this one which is new :+1: